modules/Module.php

```diff
@@ -103,8 +104,8 @@ function($event) {
         Event::on(
             Entry::class,
             Entry::EVENT_AFTER_DELETE,
-            function($event) {
-                /** @var Entry $entry */
+            function(Event $event) {
```

Comment:
I fixed this. Only some kind of _sane_ person would assume `EVENT_AFTER_SAVE` and `EVENT_AFTER_DELETE` receive the same kind of event.

Reply:
😂